controller/handler/patient.js

```diff
@@ -81,14 +81,17 @@ function patientView (request, reply) {
                 patient: processPatient(currentPatient),
                 trial: currentTrial,
                 surveys: surveyInstances.map((surveyInstance) => {
-                    surveyInstance.startTime = moment(surveyInstance.startTime).format('MM-DD-YYYY');
-                    surveyInstance.endTime = moment(surveyInstance.endTime).format('MM-DD-YYYY');
+                    surveyInstance.startTime = moment(new Date(surveyInstance.startTime)
```

Comment:
I think it already is a `Date`
but if it isn't include the format in the parse option http://momentjs.com/docs/#/parsing/string-format/
its shorter and will make our lives easier now and later :smile: